git.md

```diff
@@ -0,0 +1,35 @@
+## git
+
+### commit
+
+`git commit` saves changes
+
+Example:
+```console
+$ git commit -m "Enter here a message"
+```
+### status
+`git status`  Shows the state of working directory.
```

Comment:
There is an unneeded space before 'Show'.  Make it as a sentence: `git status` shows the state ...

Reply:
Fixed